bluemira/equilibria/coils/_grouping.py

```diff
@@ -800,10 +858,15 @@ def __init__(
 
         self.modify_symmetry(symmetry_line)
         diff = self._symmetrise()
-        self._coils[1].x = self._coils[1].x - diff[0]
-        self._coils[1].z = self._coils[1].z - diff[1]
+        self.symmetric_group.x -= diff[0]
+        self.symmetric_group.z -= diff[1]
```

Comment:
I'm sure you've checked this (as it works) but modification in place now works for positions in all cases (eg a symmetric circuit with more than one coil in each group)?

Or should we disable that usecase?

Reply:
I dono, it's the same thing that was happening before.

I think we talked about it and I think if it's a coilgroup, then if say diff[1] is a single float, coils are moved st. diff[1] is at the center otherwise there need to be enough values in diff[1] for it to set the z's for every coil